.bazelrc

```diff
@@ -67,4 +67,8 @@ build --incompatible_strict_action_env
 #build --remote_cache=https://storage.googleapis.com/megaboom-bazel-artifacts
 #build --remote_cache_compression=true
 
+# Tune these to fit our needs.
+build --experimental_disk_cache_gc_max_size=100G
```

Comment:
"By default, public shared Github runners come with around 25-29 GB of free disk space to be consumed by your build job." so this threshold is too high.  ORFS itself is going to need a bunch of that space.

Reply:
So then we might as well leave it at 100gbyte and wait for a more suitable runner?